clinica/pipelines/pet_surface/pet_surface_pipeline.py

```diff
@@ -315,6 +319,24 @@ def build_core_nodes(self):
 
         full_pipe.inputs.matscript_folder_inverse_deformation = os.path.abspath(os.path.dirname(os.path.realpath(__file__)))
 
+        # This section of code determine wether to use SPM standalone or not
+        full_pipe.inputs.use_SPM_standalone = False
+        if all(elem in os.environ.keys() for elem in ['SPMSTANDALONE_HOME', 'MCR_HOME']):
+            if os.path.exists(os.path.expandvars('$SPMSTANDALONE_HOME')) and os.path.exists(os.path.expandvars('$MCR_HOME')):
+                print(Fore.GREEN + 'SPM standalone has been found and will be used in this pipeline' + Fore.RESET)
+                if platform.system() == 'Darwin':
+                    matlab_cmd = ('cd ' + os.path.expandvars('$SPMSTANDALONE_HOME') + ' && ' + './run_spm12.sh'
+                                  + ' ' + os.environ['MCR_HOME']
+                                  + ' script')
+                elif platform.system() == 'Linux':
+                    matlab_cmd = (os.path.join(os.path.expandvars('$SPMSTANDALONE_HOME'), 'run_spm12.sh')
+                                  + ' ' + os.environ['MCR_HOME']
+                                  + ' script')
+                spm.SPMCommand.set_mlab_paths(matlab_cmd=matlab_cmd, use_mcr=True)
+                full_pipe.inputs.use_SPM_standalone = True
+            else:
+                raise FileNotFoundError('$SPMSTANDALONE_HOME and $MCR_HOME are defined, but linked to non existent folder ')
+
```

Comment:
Naive question: I have the feeling that these lines will be reused at least in T1Volume* and PETVolume pipelines. Would it be possible to factorize these resources into a function?

Reply:
I understand. I will add a card regarding this risky point.

In that case, could you create a `get_tpm()` function in `pet_surface_utils.py`? This will improve the readability of your code, decrease the complexity and will also help the person who will harmonize code regarding SPM-Standalone.

Reply:
`get_tpm()` has been corrected in an other PR, (this commit https://github.com/aramis-lab/clinica/pull/35/commits/f7281fd042539bbae66ab8aeb64743b7fe6cc208)
I'll update the pipeline once the remove_grabbit PR is merged